website/src/index.tsx

```diff
@@ -11,10 +10,10 @@ import("./errorLogging")
     // don't care if error logging fails to load
   });
 
-ReactDOM.render(<App />, document.getElementById("root"));
-
-// Strict mode is unfortuanately not used due to a reliance on the react-helmet which has not yet
-// made their code strict-safe.
+import { createRoot } from "react-dom/client";
+const container = document.getElementById("root");
+const root = createRoot(container!);
+root.render(<App />);
```

Comment:
https://reactjs.org/blog/2022/03/08/react-18-upgrade-guide.html#updates-to-client-rendering-apis